.gitignore

```diff
@@ -13,7 +13,7 @@
 
 # misc
 .DS_Store
-.env
+# .env
```

Comment:
This should be present in gitignore, any reason for commenting it out?

you can just add the vars in vercel itself

Reply:
I have added JWT secret in `.env` file and since `env` is added in gitignore, the authentication functionality was not working. Please suggest if there is any alternative

Reply:
pinged you in discord